enhancements/authentication/direct-use-of-oidc.md

```diff
@@ -0,0 +1,450 @@
+---
+title: direct-use-of-oidc
+authors:
+  - "@stlaz"
+reviewers:
+  - "@s-urbaniak"
+  - "@slaskawi"
+  - "@deads2k"
+approvers:
+  - "@deads2k"
+api-approvers: []
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+tracking-link: []
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# Direct use of OIDC as an identity provider
+
+## Summary
+
+When users authenticate to OpenShift, the authentication goes through the `oauth-server`
+which serves as middle-man actor that unifies access to multiple kinds of
+identity providers, such as LDAP, OIDC providers, providers returning HTTP Basic
+Authentication challenges, and similar.
+
+The above differs from vanilla Kubernetes where authentication is dealt directly
+by the `kube-apiserver`. This has limitations especially when it comes to being
+able to login in a uniform manner. On the other hand, `kubectl`, the binary for
+accessing Kubernetes APIs, has evolved through the years to simplify the login
+process to Kubernetes clusters, and communities built useful toolings around
+the binary.
+
+## Motivation
```

Comment:
I think one of our primary motivation is allowing oidc providers to see the gaps in the power they provide to openshift clusters as opposed to considering this a viable replacement. This is necessary to provide in order for the gaps to become obvious to the oidc providers so they can address the shortcomings.